habitat-lab/habitat/config/benchmark/rearrange/ovmm.yaml

```diff
@@ -0,0 +1,76 @@
+# @package _global_
+
+defaults:
+  - /habitat: habitat_config_base
+  - /habitat/simulator/agents@habitat.simulator.agents.main_agent: rgbdp_head_rgb_third_agent
+  - /habitat/task/rearrange: ovmm
+  - /habitat/dataset/rearrangement: floorplanner
```

Comment:
Are we changing the name to hssd or pushing that to a future pr?

Reply:
Update this particular thing to hssd. There are still other places that need updating. Will do those later